base_report_to_printer/report.py

```diff
@@ -28,9 +28,12 @@ class Report(models.Model):
     def print_document(self, cr, uid, ids, report_name, html=None,
                        data=None, context=None):
         """ Print a document, do not return the document file """
-        document = super(Report, self).get_pdf(cr, uid, ids, report_name,
-                                               html=html, data=data,
-                                               context=context)
+        if context is None:
+            context = self.pool['res.users'].context_get(cr, uid)
```

Comment:
@clonedagain please refer to mailing list https://lists.launchpad.net/openerp-community/msg06816.html.